R/user_helpers.R

```diff
@@ -123,6 +123,6 @@ print.adam_templates <- function(x, ...) {
     cat("No ADaM templates available in package '", pkg, "'\n", sep = "")
   } else {
     cat("Existing ADaM templates in package '", pkg, "':\n", sep = "")
-    cat(paste0("\U2022 ", x), sep = "\n")
+    cat(paste0("- ", x), sep = "\n")
```

Comment:
```suggestion
    bullet <- if (is.na(iconv("\U2022"))) "-" else "\U2022"
    cat(paste(bullet, x), sep = "\n")
```

Reply:
This will ensure that the better looking unicode bullet point is printed when this encoding is available and a simple `-` otherwise.

You can check for yourself using this code:

```r
x <- "ADSL"
bullet <- if (is.na(iconv("\U2022"))) "-" else "\U2022"
cat(paste(bullet, x), sep = "\n")

Sys.setlocale(category = "LC_ALL", locale = "C")
bullet <- if (is.na(iconv("\U2022"))) "-" else "\U2022"
cat(paste(bullet, x), sep = "\n")
```

Kudos to @dgkf for showing me this!